istio/assets/configuration/spec.yaml

```
@@ -0,0 +1,52 @@
+name: Istio
+files:
+  - name: istio.yaml
+    options:
+    - template: init_config
+      options:
+        - template: init_config/default
+        - template: init_config/openmetrics
+    - template: instances
+      options:
+        - name: istio_mesh_endpoint
+          description: |
+            To enable Istio metrics you must specify the url exposing the API.
+            Note for RHEL and SUSE users: due to compatibility issues, the check does not make use of
+            the CPP extension to process Protocol buffer messages coming from the api. Depending
+            on the metrics volume, the check may run very slowly.
+          value:
+            example: http://istio-telemetry.istio-system:42422/metrics
```

https://datadoghq.dev/integrations-core/meta/config_specs/#example-file-consumer

```suggestion
            default: null
            example: http://istio-telemetry.istio-system:42422/metrics
```